packages/grpc/src/internal/client.ts

```diff
@@ -86,7 +88,7 @@ export function createGrpcClient<T>(
 
         const metaCallback = (_params: any, callback: (arg0: null, arg1: Metadata) => void) => {
             const meta = new Metadata();
-            meta.add("custom-auth-header", "token");
+            meta.add("custom-auth-header", apiKey || DEFAULT_API_KEY);
```

Comment:
I think we can skip adding the header if there is no API Key provided. Also header name should be "authorization"
```suggestion
if(apiKey) {
            meta.add("authorization", apiKey);
            }
```

Reply:
Added the if check a bit earlier